platform/ios/src/MGLMapView.mm

```diff
@@ -4736,153 +4737,167 @@ - (void)unrotateIfNeededAnimated:(BOOL)animated
     }
 }
 
-- (void)notifyMapChange:(mbgl::MapChange)change
-{
-    // Ignore map updates when the Map object isn't set.
+- (void)onRegionWillChange:(bool)animated {
```

Comment:
`-cameraWillChangeAnimated:` would also be acceptable.

The macOS SDK implementation of MGLMapViewDelegate already uses the term “camera” instead of “region”. The iOS SDK implementation blindly copied MapKit’s use of “region” without bringing over MapKit’s `MKRegion` struct.

We’ll want to fix the public interface when we’re ready for backwards-incompatible changes, but for now, either name works for this internal method.